.gitmodules

```diff
@@ -25,3 +25,30 @@
 [submodule "terraform/modules/providers/aws/virtual-machine-storage"]
 	path = terraform/modules/providers/aws/virtual-machine-storage
 	url = https://github.com/onaio/terraform-aws-ebs-volume.git
+[submodule "roles/onaio.gpg-import"]
+	path = roles/onaio.gpg-import
+	url = git@github.com:onaio/ansible-gpg-import.git
```

Comment:
@morrismukiri please use the https URLs for the submodules. I think this is needed since this is an open source project (and likely needed by someone who doesn't have a GitHub account).